src/conf/security/generate_openvex.py

```diff
@@ -0,0 +1,177 @@
+#!/usr/bin/env python3
```

Comment:
Hi @ppkarwasz 
This seems like a bad idea because all 20+ components will need this duplicated. It seems we should have an "empty" VEX statement for components without issues to affirm that we are OK there. We have TWO plug-ins already for this kind of housekeeping (we really should have a single one), can't we stick stuff like this in there?